requirements/all.txt

```diff
@@ -5,7 +5,7 @@
 #conda: esmpy>=7.0  (only python=2)
 #gdal : under review -- not tested at present
 mo_pack
-nc-time-axis  # conda: nc_time_axis
+nc-time-axis  #conda: nc_time_axis
```

Comment:
This could be dropped now. I just merged te rename :smile:

Reply:
Nice!